src/main/java/de/retest/recheck/ignore/CacheFilter.java

```diff
@@ -37,10 +38,19 @@ public boolean matches( final Element element, final AttributeDifference attribu
 		return attributeCache.computeIfAbsent( Pair.of( element, attributeDifference ), matchesDiff() );
 	}
 
+	@Override
+	public boolean matches( final Element element, final String attributeKey ) {
+		return attributeKeyCache.computeIfAbsent( Pair.of( element, attributeKey ), matchesKeyDiff() );
+	}
+
```

Comment:
I would move this method below, so that each computation function is below its `matches` function.